Watchdog/form.json

```diff
@@ -1,17 +1,67 @@
 {
   "elements":
   [
-	{ "type": "Label", "label": "Time until not updatet objects will set off an alert. Its added to the last time an object is updatet." },
-	{ "type": "Select", "name": "TimeBase", "caption": "Time base",
-		"options": [
-			{ "label": "Seconds", "value": 0 },
-			{ "label": "Minutes", "value": 1 },
-			{ "label": "Hours", "value": 2 },
-			{ "label": "Days", "value": 3 }
+	{
+		"type": "List",
+		"name": "Targets",
+		"caption": "Variables",
+		"add": true,
+		"delete": true,
+		"columns": [
+		  {
+			"caption": "Variable",
+			"name": "VariableID",
+			"width": "auto",
+			"add": 0,
+			"edit": {
+			  "type": "SelectVariable"
+			}
+		  },
+		  {
+			"caption": "Name (optional)",
```

Comment:
Auch Elemente in die locale.json aufzunehmen, die im deutschen und englischen gleich sind